README.md

```diff
@@ -1,214 +1,201 @@
-`pnpm --filter "./packages/** <script>`.
+1. Install the CLI globally:
 
-For example:
+   ```bash
+   npm install -g @openfn/cli
+   ```
 
-```
-pnpm --filter "./packages/**" build
-pnpm --filter "./packages/**" test
-```
+2. Create a job file (e.g., `job.js`):
 
-## Contributing
+   ```javascript
+   // Example: Send data to Salesforce
+   create('Account', {
+     Name: $.data.company_name,
+     Industry: $.data.industry,
+   });
+   ```
 
-### Assign yourself to an issue
+3. Run the job:
+   ```bash
+   openfn job.js -a salesforce -s tmp/state.json -o tmp/output.json
+   ```
 
-Read through the existing [issues](https://github.com/OpenFn/adaptors/issues)
-and assign yourself to the issue you have chosen. If anything needs
-clarification, don't hesitate to leave a comment on the issue and we will get
-back to you as soon as possible.
+### With OpenFn Lightning
 
-If there isn't already an issue for the feature you would like to contribute,
-please make one and assign yourself.
+Lightning is our open-source workflow automation platform that uses these
+adaptors. See the
+[Lightning documentation](https://github.com/OpenFn/lightning/blob/main/RUNNINGLOCAL.md#using-local-adaptors)
+for setup and usage instructions.
 
-### Open a pull request
+## Contributing
 
-1. Clone the adaptors repository, then
-   [fork it](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+1. **Find an issue**: Browse
+   [open issues](https://github.com/OpenFn/adaptors/issues) and assign yourself
+   to one, or create a new issue for your feature.
 
-2. Make your changes by following the
-   [working with adaptors](#working-with-adaptors) section.
+2. **Fork and clone**: Fork the repository and clone your fork locally.
 
-3. Open a draft PR, fill out the pull request template (this will be added
-   automatically for you) then make sure to self-review your code and go through
-   the 'Review checklist'. Leave any notes for the reviewer in the 'Details'
-   section.
+3. **Make changes**: Follow the
+   [development workflow above](#development-steps).
 
-4. Mark the pull request as ready for review, and assign @stuartc or
-   @taylordowns2000.
+4. **Submit PR**: Create a draft PR, fill out the template, self-review, then
+   mark as ready for review and assign @mtuchi or @josephjclark.
 
-### Working with adaptors
+## Development
 
-#### Requirements
+### Creating a New Adaptor
 
-- Create a fork of this repo on your local machine
-- Iinstall the OpenFn CLI `npm install -g @openfn/cli`
-- Ensure the OPENFN_REPO_DIR env var is set to to the root of this repo, ie,
-  `export OPENFN_REPO_DIR=~/repo/openfn/cli-repo`
-- Ensure you have the logo images of the adaptor you need to create: a rectangle
-  512x190px and a square-256x256px
+#### Prerequisites
 
-#### 1. Setup
+1. Set up the Repository Directory
 
-Generating a new adaptor. Run this from the repo root:
+   The CLI needs a designated folder to save and load adaptors. Configure this
```

Comment:
Well, to be fair, `OPENFN_REPO_DIR` has a sensible default. Most users actually don't need to set this explicitly - it's just a convenience for those who want to organise their system differently.

I think we need to document both env vars somwhere (particularly the monorepo one). But I don't think it's a pre-requisite.